nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,145 @@
+//import
+const express = require("express");
+const router = express.Router();
+const meals = require("./../data/meals.json");
+
+router.get("/", async (req, res) => {
+  let data = meals;
+
+  if ("date" in req.query ) {
+    try {
+      data = data.filter((meal) =>
+    new Date(meal.createdAt).getTime() > new Date(req.query.date).getTime());
```

Comment:
hi Stepohen.I tested this and it works fine. If I only use the new Date it will return the YYYY-MM-DD of this format but ignore the time. that is why I use getTime().